app/boards/shields/jorne/boards/nice_nano.overlay

```diff
@@ -0,0 +1,29 @@
+&spi1 {
+	compatible = "nordic,nrf-spi";
```

Comment:
```suggestion
	compatible = "nordic,nrf-spim";
```
We'll probably be making this change to all boards soon. Normal SPI _seems_ to have some issues. Thanks!

Reply:
Done.